src/commands/buildCommands/status.js

```diff
@@ -0,0 +1,119 @@
+const { promisify } = require('util');
+const request = require('request');
+
+// hard-coded to .com, not .org - Travis is migrating everyting to .com
+// but in the interim this script will not work with travis-ci.org builds
+const TRAVIS_API_URL = 'https://api.travis-ci.com';
+
+const _get = promisify(request.get);
+const _post = promisify(request.post);
+const get = (path, options = {}) => _get(`${TRAVIS_API_URL}${path}`, options);
+const post = (path, options = {}) => _post(`${TRAVIS_API_URL}${path}`, options);
+
+// Travis API needs auth token and repo to search for data
+// https://developer.travis-ci.com/
+const getTravisApi = ({ token, repo }) => {
+	const headers = {
+		'Travis-API-Version': 3,
+		Authorization: `token ${token}`,
+	};
+	return {
+		build: {
+			cancel: id =>
+				post(`/build/${id}/cancel`)
+					.then(({ body }) => {
+						console.log(`Canceled build ${id}`);
```

Comment:
why `console.log` vs `console.info`?

Reply:
because this is a log of what happened.

`console.info` doesn't provide any useful distinction because from Travis's perspective it's all just `stdout` - there's no way to filter by log level, so the 'default' `console.log` is fine.